src/Modules/UngradedModules/FreeModElem.jl

```diff
@@ -172,7 +180,8 @@ Return the `i`th basis vector of `F`, that is, return the `i`th standard unit ve
 """
 function basis(F::AbstractFreeMod, i::Int)
   @assert 0 < i <= ngens(F)
-  s = Hecke.sparse_row(base_ring(F), [(i, base_ring(F)(1))])
+  e = one(base_ring(F))
+  s = sparse_row(base_ring(F), [i], [e]; sort=false)
```

Comment:
```suggestion
  s = sparse_row(base_ring(F), [i], [e])
```
no longer needed due to https://github.com/thofma/Hecke.jl/pull/1502